apps/dashboard/components/logs/table-action.popover.tsx

```diff
@@ -103,13 +103,8 @@ export const KeysTableActionPopover = ({
           {items.map((item, index) => (
             <div key={item.id}>
               <div className="px-2">
-                {/* biome-ignore lint/a11y/useKeyWithClickEvents: <explanation> */}
-                <div
-                  ref={(el) => {
-                    if (el) {
-                      menuItems.current[index] = el;
-                    }
-                  }}
+                <button
+                  type="button"
```

Comment:
good change -- semantic elements without biome overrides is usually the best choice